src/js/components/tasks/table.tsx

```diff
@@ -121,6 +125,7 @@ const AssigneeTableCell = ({
   openAssignProjectUsersModal,
   assignUserAction,
   item,
+  className,
```

Comment:
nor here, lol

Reply:
Yeah, this is another case where we're not using the class anyway.